src/D2L.Security.OAuth2/Keys/Default/D2LSecurityToken.Disposal.cs

```diff
@@ -31,13 +31,6 @@ partial class D2LSecurityToken : IDisposable {
 					if( ShouldDispose() ) {
 						m_disposed = true;
 
-						foreach( var key in m_key.Values ) {

```

Comment:
Will have to look into this, given the `SecurityKey` themselves still aren't `IDisposable`